styles/native/ts/core/widgets/linechart.ts

```diff
@@ -0,0 +1,92 @@
+import { brand, font } from "../variables";
+import { LineChartType } from "../../types/widgets";
+
+/*
+
+DISCLAIMER:
+Do not change this file because it is core styling.
+Customizing core files will make updating Atlas much more difficult in the future.
+To customize any core styling, copy the part you want to customize to styles/native/app/ so the core styling is overwritten.
+
+==========================================================================
+    Line Chart
+
+    Default Class For Mendix Line Chart Widget
+========================================================================== */
+// eslint-disable-next-line @typescript-eslint/camelcase
+export const com_mendix_widget_native_linechart_LineChart: LineChartType = {
+    container: {
+        // All ViewStyle properties are allowed
+    },
+    errorMessage: {
+        // All TextStyle properties are allowed
+        fontFamily: font.family,
+        fontSize: font.sizeSmall,
+    },
+    chart: {
+        // All ViewStyle properties are allowed
+    },
+    gridAndLabelsRow: {
+        // All ViewStyle properties are allowed
+    },
+    gridRow: {
+        // All ViewStyle properties are allowed
+    },
+    gridWrapper: {
+        // All ViewStyle properties are allowed
+    },
+    grid: {
+        padding: { top: 8, right: 8, bottom: 32, left: 32 },
+        xAxis: {
+            grid: { stroke: "#CED0D3" },
+            axis: { stroke: "#CED0D3" },
+            tickLabels: { fontFamily: font.family, fontSize: font.sizeSmall },
+        },
+        yAxis: {
+            grid: { stroke: "#CED0D3" },
+            axis: { stroke: "#CED0D3" },
+            tickLabels: { fontFamily: font.family, fontSize: font.sizeSmall },
+        },
+    },
+    xAxisLabel: {
```

Comment:
Imo it makes more sense to add this inside `xAxis` & `yAxis`. Because we have 2 different labels my suggestion for the names are `innerLabel` and `outerLabel`

Reply:
Agree, whats the difference between `innerLabel` and `outerLabel` ? Can't we just have one prop `axisLabel`

Reply:
Moved this label inside the xAxis style property